### PR TITLE
[CDAP-13376][CDAP-13825] Support icons for provisioner

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: 
- https://issues.cask.co/browse/CDAP-13376
- https://issues.cask.co/browse/CDAP-13825

### CODE REVIEW

File: cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/ProvisionerSelection/ProvisionerSelection.scss [hunk 1]
```diff
@@ -34,33 +34,30 @@ $icon-size: 93px;
...
         &:hover {
           text-decoration: none;
-          background-color: $grey-08;
-
-          .provisioner-icon {
-            > img {
-              background-color: $grey-08;
-            }
-          }
+          box-shadow: 1px 2px 10px 0 #3b78e7;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/ProvisionerSelection/ProvisionerSelection.scss]: does the shadow color exist in variables?

File: cdap-ui/app/cdap/components/ExperimentalBanner/ExperimentalBanner.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+@import '../../styles/variables.scss';
+
+.experimental-banner {
+  width: 47px;
+  display: inline-block;
+  position: relative;
+  color: white;
+  background-color: $brand-primary-color;
+  text-shadow: 0px 1px 2px #bbbbbb;
+  box-shadow: 0px 2px 4px #888888;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/ExperimentalBanner/ExperimentalBanner.scss]: use variables?

**Comment by tonybach** [comment]
Green build: https://builds.cask.co/browse/CDAP-URUT49-2

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Pull request merged**